wiki-service/src/main/java/org/exoplatform/wiki/service/rest/NotesRestService.java

```diff
@@ -99,28 +100,8 @@ public NotesRestService(WikiService wikiService, ResourceBundleService resourceB
     cc.setNoStore(true);
   }
 
-  /**
-   * Return the note page content as html or wiki syntax.
-   * 
-   * @param text contain the data as html
-   * @return the instance of javax.ws.rs.core.Response
-   * @LevelAPI Experimental
-   */
-  @POST
-  @Path("/content/")
-  public Response getNotePageContent(@FormParam("text") String text) {
-    try {
-      String outputText = WikiHTMLSanitizer.markupSanitize(text);
-
-      return Response.ok(outputText, MediaType.TEXT_HTML).cacheControl(cc).build();
-    } catch (Exception e) {
-      log.error("Error while converting wiki page content: " + e.getMessage(), e);
-      return Response.serverError().entity(e.getMessage()).cacheControl(cc).build();
-    }
-  }
-
   @GET
-  @Path("/{wikiType}/{wikiOwner:.+}/pages/{pageId}")
+  @Path("/pages/{wikiType}/{wikiOwner:.+}/{pageId}")
```

Comment:
should be /notes

Reply:
/notes is the path of the nodes's rest service here we added /pages referring to the data type (we may have rather calls for other data types as comments, attachments, wikis...)
So in this case the rest call of this method is ../rest/nodes/pages/{wikiType}/{wikiOwner:.+}/{pageId}

Reply:
then I would prefer /notes/note/

page was the name for wikis. we changed the terminology. Now it’s notebooks for wikis and notes for page

Reply:
Ok, I'll change it